config/src/main/java/com/quorum/tessera/config/ServerConfig.java

```diff
@@ -48,20 +48,25 @@
     @XmlElement
     private String serverAddress;
 
+    @XmlElement
+    private String corsDomains;
+
     public ServerConfig(final AppType app,
                         final boolean enabled,
                         final String serverAddress,
                         final CommunicationType communicationType,
                         final SslConfig sslConfig,
                         final InfluxConfig influxConfig,
-                        final String bindingAddress) {
+                        final String bindingAddress,
+                        final String corsDomains) {
```

Comment:
I'd also be tempted to not set this in constructor and default to null value. Will save having to change existing tests.